BaseFilter/sections/specific.txt

```diff
@@ -277,6 +277,7 @@ chortle.co.uk,coloring.ws,dltk-holidays.com,dltk-kids.com,kidzone.ws,pcsteps.com
 !
 !
 !
+alt-codes.net###copyModal .modal-body
```

Comment:
Looks ok, but there is no tag in `.modal-body`. I see only `div.modal-body`.

Reply:
Yes. But, I added it for additional safety.